src/av2/rendering/map.py

```diff
@@ -125,15 +125,15 @@ def render_lane_boundary_egoview(
                 dash_interval_m=DASH_INTERVAL_M,
             )
 
-        elif (mark_type == LaneMarkType.SOLID_DASH_YELLOW and side == "right") or (
+        elif (mark_type in [LaneMarkType.SOLID_DASH_WHITE, LaneMarkType.SOLID_DASH_YELLOW] and side == "right") or (
```

Comment:
Thanks for the PR, @johnlambert-argo. What are your thoughts on defining the valid line types as:

`VALID_MARK_TYPES = set([LaneMarkType.SOLID_DASH_WHITE, LaneMarkType.SOLID_DASH_YELLOW])` (or whichever variable name is suitable)

and then reusing this set on line 128 and 136?